src/node_data.cpp

```diff
@@ -13,7 +13,7 @@
 
 namespace YAML {
 namespace detail {
-std::atomic<size_t> node::m_amount{0};
+YAML_CPP_API std::atomic<size_t> node::m_amount{0};
```

Comment:
Isn't same YAML_CPP_API should be in header too?

Reply:
No, in the definition is enough, for this internal symbol.